lib/ansible/modules/messaging/kafka_configs.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Guillaume Delpierre <gde@llew.me>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: kafka_configs
+author: "Guillaume Delpierre (@gdelpierre)"
+version_added: "2.5"
+
+short_description: Manage entity configs.
+description:
+    - 'Add/Remove entity config for a topic, client, user or broker'
+requirements:
+    - 'kafka >= 2.11-0.10'
+options:
+    configs:
+        description:
+            - Key / Value (str) pairs of configs.
+    describe:
+        description:
+            - List configs for the given entity.
+    entity_default:
+        description:
+            - Default entity name for clients/users.
+    entity_name:
+        description:
+            - Name of entity.
+    entity_type:
+        required: True
+        description:
+            - Type of entity.
+    executable:
+        description:
+            - Kafka executable path if not in your current PATH or if it name differs.
+    zookeeper:
+        required: True
+        description:
+            - The connection string for the zookeeper connection.
+              Multiple connection strings are allowed.
+'''
+
+EXAMPLES = '''
+name: 'list config'
```

Comment:
```suggestion
- name: 'list config'
```

Same for others. You will need to check indentation